src/components/UI/BookComponent/BookComponent.module.css

```diff
@@ -0,0 +1,49 @@
+.book {
+  display: flex;
+  width: 128px;
+  flex-direction: column;
+  box-shadow: none;
+  gap: 8px;
+}
+.cover {
+  border-radius: 10px;
+  height: 192px;
+  width: 100%;
+  position: relative;
+}
+.content {
+  max-width: 100%;
+  gap: 4px;
+}
+.title {
+  color: var(--dark-primary-color);
+  text-overflow: ellipsis;
```

Comment:
for the ellipsis to be active the item must have a max width with pixels, did you try it?

Reply:
No, I didn't but the ellipsis worked fine, I'll add it